src/Kliva/ViewModels/MainViewModel.cs

```diff
@@ -96,32 +96,10 @@ public ActivitySummary SelectedActivity
         public RelayCommand SettingsCommand => _settingsCommand ?? (_settingsCommand = new RelayCommand(() => NavigationService.Navigate<SettingsPage>()));
 
         private RelayCommand<ActivitySummary> _kudosCommand;
-        public RelayCommand<ActivitySummary> KudosCommand => _kudosCommand ?? (_kudosCommand = new RelayCommand<ActivitySummary>(async (x) => await OnKudosAsync(x)));
+        public RelayCommand<ActivitySummary> KudosCommand => _kudosCommand ?? (_kudosCommand = new RelayCommand<ActivitySummary>(OnKudosAsync));
 
         private RelayCommand<ActivitySummary> _commentCommand;
-        public RelayCommand<ActivitySummary> CommentCommand => _commentCommand ?? (_commentCommand = new RelayCommand<ActivitySummary>(async (x) => await OnCommentAsync(x)));
-
-        private async Task OnKudosAsync(ActivitySummary activitySummary)
-        {
-            // make sure we work with the swiped item
-            // this could happen if we use the mouse to swipe and move the mouse outside the ListView and release it there
-            if (activitySummary != this.SelectedActivity)
-            {
-                this.SelectedActivity = activitySummary;
-            }
-            MessengerInstance.Send<ActivitySummaryKudosMessage>(new ActivitySummaryKudosMessage(_selectedActivity));
-        }
-
-        private async Task OnCommentAsync(ActivitySummary activitySummary)
-        {
-            // make sure we work with the swiped item
-            // this could happen if we use the mouse to swipe and move the mouse outside the ListView and release it there
-            if (activitySummary != this.SelectedActivity)
-            {
-                this.SelectedActivity = activitySummary;
-            }
-            MessengerInstance.Send<ActivitySummaryCommentMessage>(new ActivitySummaryCommentMessage(_selectedActivity));
-        }
+        public RelayCommand<ActivitySummary> CommentCommand => _commentCommand ?? (_commentCommand = new RelayCommand<ActivitySummary>(OnCommentAsync));
```

Comment:
Jan, Bart meant that you created 2 methods called OnKudosAsync and OnCommentAsync, but the code in these 2 methods don't contain any Async references... You only call MessengerInstance.Send<...>

So his suggestion : Rename the methods to OnKudos and OnComment!